<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Money</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="money_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="money_scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>455</width>
            <height>503</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>NHS current fees (money0)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="money0_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>NHS payments (money2)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="money2_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>NHS estimated (money4)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="money4_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>private estimate (money5)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="money5_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Exempt - nhs gross - completed treatment (money6)</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="money6_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Exempt - NHS gross - estimated current (money7)</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="money7_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>credit (money8)</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QSpinBox" name="money8_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>debt (money9)</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QSpinBox" name="money9_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>debt2 (money10)</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QSpinBox" name="money10_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>money 11</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QSpinBox" name="money11_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>private current fees (money1) </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="money1_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="money3_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>private payments (money3)</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>ALL AMOUNTS ARE IN PENCE (cents)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Outstanding amount</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="total_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Dates</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="2">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>455</width>
            <height>520</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Last CE (pd5)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QDateEdit" name="pd5_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd5_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Last ECE (pd6)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDateEdit" name="pd6_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd6_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Last FCA (pd7)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDateEdit" name="pd7_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd7_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Last OPT (pd8)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDateEdit" name="pd8_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd8_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Last intraoral Xrays (pd9)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QDateEdit" name="pd9_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd9_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Last SP (pd10)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QDateEdit" name="pd10_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pd10_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Last Account sent (billdate)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QDateEdit" name="billdate_dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="billdate_pushButton">
               <property name="text">
                <string>Add Date</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Miscellaneous</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Hidden Notes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="hidden_notes_tableWidget">
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
